indra/newview/llviewertexturelist.cpp

```diff
@@ -561,9 +561,9 @@ LLViewerFetchedTexture* LLViewerTextureList::getImage(const LLUUID &image_id,
                                                    LLHost request_from_host)
 {
     LL_PROFILE_ZONE_SCOPED_CATEGORY_TEXTURE;
-    if(!mInitialized)
```

Comment:
Crash time: 2024-07-20T22:49:39Z (according to plCrashReporter.crashlog)
Log time: 2024-07-20T23:03:30Z
This looks like a log from a different session.

Why do you think isExiting() will fix it?
Call hapens from withing (!LLApp::isExiting()) check and if error was set by something texture cach should still be valid.